cu-webteam/d8-platform
# Image alt tag fix

# Description

Adding the _header background image_, _news/spotlight_ and making the _alt tag_ required.

## Related PRs

List related PRs against other branches:

- https://github.com/cu-webteam/d8-platform/pull/56

## Steps to Test or Reproduce

Create a header background image and news/spotlight image. Ensure the title and alt are present when viewed on the front end. Also sure ensure when creating that the 'alt' tag is required.

```shell
```
